faqs/basics.md

```diff
@@ -68,3 +68,17 @@ When you buy and stake OHM, you capture a percentage of the supply (market cap)
 ### What is a rebase?
 
 Rebase is a mechanism by which your staked OHM balance increases automatically. When new OHM are minted by the protocol, a large portion of it goes to the stakers. Because stakers only see staked OHM balance instead of OHM, the protocol utilizes the rebase mechanism to increase the staked OHM balance so that 1 staked OHM is always redeemable for 1 OHM.
+
+### What is APY?
+
+APY stands for annual percentage yield. It measures the real rate of return on your principal by taking into account the effect of compounding interest. In the case of OlympusDAO, your staked OHM represents your principal, and the compound interest is added periodically on every epoch (2200 Ethereum blocks, or around 8 hours) thanks to the rebase mechanism.
```

Comment:
It says `the real rate` but it will by no means be the real rate people get. I would like to phrase it in a way that makes people realize that the stated APY is a rough estimation that might change over the course of a year. The only thing that is for certain is the reward rate in the distributor contract.

Reply:
I intend it to be a general definition for APY, hence the real rate of return. The accompanying graphic also assumes a fixed APY, so I think we shouldn't mention about the fluctuating APY here. How about I add another entry like "How is the APY calculated?"

Reply:
It would be cool to have a section for the real calculation and how reward rate -> OHM distributed -> APY works.